src/diffusers/models/attention.py

```diff
@@ -148,6 +148,7 @@ def __init__(
         attention_type: str = "default",
         positional_embeddings: Optional[str] = None,
         num_positional_embeddings: Optional[int] = None,
+        _explicitly_mark_as_cross_attention: Optional[bool] = False,
```

Comment:
why do we need this?

Reply:
Because PixArt Alpha doesn't use different dimensions for cross-attention. So, the existing condition to detect if we're on a cross-attention block breaks.

Reply:
Can't we get it to work by doing: https://github.com/huggingface/diffusers/pull/6181/files#r1436538609 ?